app/src/main/java/com/gh4a/fragment/EventListFragment.java

```diff
@@ -368,15 +366,28 @@ public void onCreateContextMenu(ContextMenu menu, View v, ContextMenuInfo menuIn
 
         } else if (Event.TYPE_ISSUES.equals(eventType)) {
             IssuesPayload payload = (IssuesPayload) event.getPayload();
-            int issueNumber = payload.getIssue().getNumber();
+            final int issueNumber = payload.getIssue().getNumber();
             menu.add(getString(R.string.menu_issue, issueNumber))
-                    .setIntent(IssueActivity.makeIntent(getActivity(), repoOwner, repoName, issueNumber));
+                    .setOnMenuItemClickListener(new MenuItem.OnMenuItemClickListener() {
```

Comment:
This whole if block should get refactored. I also want to add some missing action and improve naming. But that's for another pull request so I think it's ok to leave these 2 new listeners like this for now.